Heapsort

```diff
@@ -0,0 +1,79 @@
+void heapsort(int A[],int n);
```

Comment:
Please include #include <stdio.h>. Even though the compiler takes this for granted this days in PCs it might not be same for all. Since you are using printf()